hack/verify-generated-crd-code.sh

```diff
@@ -17,3 +17,13 @@
 HACK_DIR=$(dirname "${BASH_SOURCE}")
 
 ${HACK_DIR}/update-generated-crd-code.sh --verify-only
```

Comment:
Should the `--verfify-only` flag here be removed, since it's not being used anymore?

Reply:
@nrb since adnan's out for a week, do you want to merge this as-is now and do a followup to remove that flag? would love to get it out of the queue :)